docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -332,7 +332,7 @@ For the full list of the supported expression, please check the link:https://pkg
 +
 [source,bash]
 ----
-kubectl apply -f ./hot-backup-agent-scheduled.yaml
+kubectl apply -f ./cron-hot-backup-agent.yaml
 ----
 
 === Checking the Status of a Backup
```

Comment:
```suggestion
=== Checking the Status of an External Backup
```
Can you check the status of a specific scheduled backup, i.e. by filename?

Also, could we say: "To check the status of an external backup, run the following command:"